pkg/errors/errors.go

```diff
@@ -192,3 +193,14 @@ func NewContainerError(name string, code int, output string) error {
 		ExitCode:   code,
 	}
 }
+
+// NewSourcePathError returns a new error which indicates there was a problem
+// when accessing the source code from the local filesystem
+func NewSourcePathError(path string) error {
+	return Error{
+		Message:    fmt.Sprintf("Local filesystem source path does not exist: %s", path),
+		Details:    nil,
+		ErrorCode:  SourcePathError,
+		Suggestion: "check the source code path on hte local filesystem",
```

Comment:
I know the pain, I'm promising myself to go to bad at reasonable time this week, but too much to do and so not much time. Do you know how to stretch a day to fit at least 48h in it :wink: